src/main/webapp/js/maps-page-loader.js

```diff
@@ -2,42 +2,58 @@ const MAP_ELEMENT_ID = 'map';
 const MAP_NIGHT_TYPE_ID = 'night'
 /**
  * This function is meant to create the map, set the centre and zoom, add a new map style and add markers.
- * Map styles are different modes that will be displayed to the user (The user can switch around with button)
- * Added night mode (the static configs for that style is in extraMapStyles.json)
+ * Map styles are different modes that will be displayed to the user
+ * Added night mode (static configs for that style in extraMapStyles.json)
  * New modes/map styles can be added in extraMapStyles.json file.
  *
  */
-function initConfigMap() {
-  $.getJSON("/json/extraMapStyles.json", function(json) {
-    // Create a new StyledMapType object, passing it an array of styles,
-    // and the name to be displayed on the map type control.
-    const styledMapType = new google.maps.StyledMapType(json.Night,{name: 'Night'});
+function fetchNightStyleJSONValues() {
+  const url = '/json/extraMapStyles.json';
+  fetch(url)
+    .then(response => response.json())
+    .then(data => addDiffStylesToMap(data.Night));
+}
 
-    // Create a map object, and include the MapTypeId to add
-    // to the map type control.
-    const map = new google.maps.Map(document.getElementById(MAP_ELEMENT_ID), {
-      center: {lat: 1.2764558, lng: 103.7996469},
-      zoom: 16,
-      mapTypeControlOptions: {
-        mapTypeIds: ['roadmap', 'satellite', 'hybrid', 'terrain', MAP_NIGHT_TYPE_ID]
-      }
-    });
+function addDiffStylesToMap(nightStyle){
```

Comment:
I was thinking maybe you can consider renaming the variable 'nightStyle' here to something a bit more general. In case you want to incoorporate more styles in the future, you will be able to reuse this function without it being tailored to fit just one specific style.

Reply:
yeap will do :)